libs/invitation/src/lib/invitation.module.ts

```diff
@@ -10,11 +10,11 @@ import { InvitationComponent } from './invitation.component';
 import { InvitationListModule } from './components/list/list.module';
 import { ImageModule } from '@blockframes/media/image/directives/image.module';
 import { StaticSelectModule } from '@blockframes/ui/static-autocomplete/select/static-select.module';
+import { LogoSpinnerModule } from '@blockframes/ui/rive/logo-spinner.module'
```

Comment:
```suggestion
import { LogoSpinnerModule } from '@blockframes/ui/rive/logo-spinner.module';
```

please look for other missing `;`